kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWriteSuiteBase.scala

```diff
@@ -137,8 +176,125 @@ trait DeltaTableWriteSuiteBase extends AnyFunSuite with TestUtils {
     Optional.empty()
   }
 
+  def generateData(
```

Comment:
Has this been copied as is?

Reply:
I just moved it from `DeltaTableWriteSuite` to `DeltaTableWriteSuiteBase` so that both `DeltaTableWriteSuite` and `InCommitTimestampSuite` can use it.